src/Sulu/Bundle/ContentBundle/Resources/public/js/components/internal-links/main.js

```diff
@@ -236,7 +237,9 @@ define([], function() {
             return templates.contentItem(
                 item[this.options.idKey],
                 item.title,
-                this.getItemIcons(item)
+                item.url,
+                this.getItemIcons(item),
+                this.sandbox.util.cropMiddle
```

Comment:
Do we win something by passing the function like this? Makes things more complicated... If we need it this way I am tempted to say that we have to check in the template function if a function was passed.